site/content/entry/ldn-is-here/index.md

```diff
@@ -0,0 +1,159 @@
+Remember to port forward, otherwise your friends won’t be able to connect!
+
+Unverified users aren’t left without options, though - they still have the ability to create unlisted rooms, direct connecting, and can join any yuzu hosted room.
+
+Do note that verified users will have their privileges revoked for violating any site policies while on the room chatroom. 
+Please respect the "recommended game" listed in publicly hosted rooms, as even unrelated games will add to the bandwidth load.
+
+{{< imgs "./room_chat.png" >}}
+
+**Note: You can use these rooms for LAN mode games too - instead of ZeroTier or Hamachi.**
+
+## Development
+
+FearlessTobi began the development by leveraging the existing virtual rooms infrastructure from [Citra](https://citra-emu.org).
+Due to code similarities between yuzu and Citra, this tried and tested infrastructure easily became the foundation of this feature.
+
+Now that the virtual rooms were taken care of, Tobi moved onto the `LDN` service and its protocols.
+Using the research and code from the [ldn_mitm](https://github.com/spacemeowx2/ldn_mitm) project, he implemented the necessary changes for yuzu instances to communicate with each other over these virtual rooms.
+
+As the feature slowly matured, Tobi began leveraging yuzu's internal team of testers to test the feature across several titles for compatibilty and performance validation. 
+And to prevent user IPs from being leaked when connected to rooms, Tobi changed the implementation to use fake IPs for routing.
+
+With the multiple testing iterations, various bugs and glitches were found. 
+[german77](https://github.com/german77) relied on his own reverse-engineering and hardware tests, to debug these issues and fixed them all.
+He also made several UI fixes to improve its usability and brought the codebase up to yuzu standards.
+
+## Compatibility
+
+Please note that as this is the initial release of the feature, some games might have some limitations.
+We hope to slowly fix these over the course of next few months.
+
+yuzu's Local Wireless Multiplayer was tested by our testers across a small set of games and their corresponding compatibilty is as follows:
+
+{{< imgs "./compat.png" >}}
+
+
+## Screenshots
+
+{{< single-title-imgs
+    "Animal Crossing: New Horizons"
+    "./acnh1.jpg"
+    "./acnh2.png"
+>}}
+
+{{< single-title-imgs
+    "Pokémon Legends: Arceus"
+    "./arceustrade1.png"
+    "./arceustrade2.png"
+>}}
+
+{{< single-title-imgs
+    "Mario Kart 8 Deluxe"
+    "./mk8d_2.png"
+	"./mk8d_3.png"
+>}}
+
+{{< single-title-imgs
+	"Super Mario Smash Bros. Ultimate"
+	"./ssbu_ldn_3.png"
+	"./ssbu_ldn_4.png"
+	"./smash_ldn.png"
+>}}
+
+{{< single-title-imgs
+	"Splatoon 2"
+	"./splatoon2_1.png"
+	"./splatoon2_2.png"
+	"./splatoon2_3.png"
+>}}
+
+{{< imgs
+	"./lm3.png| Luigi's Mansion 3"
+>}}
+
+{{< imgs
+	"./cowabunga.png| Teenage Mutant Ninja Turtles: The Cowabunga Collection" 
+	"./diablo3.png| Diablo III: The Eternal Collection"
```

Comment:
```suggestion
	"./diablo3.png| Diablo III: Eternal Collection"
```
Please retake screenshot so it includes only gameplay to match other screenshots in this section